pkgs/development/python-modules/maestral/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, python
+, sdnotify
+, systemd
+}:
+python.pkgs.buildPythonApplication rec {
+  pname = "maestral";
+  version = "1.1.0";
+
+  disabled = python.pkgs.pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "SamSchott";
+    repo = "maestral";
+    rev = "v${version}";
+    sha256 = "0d1pxbg69ll07w4bbpzs7zz1yn82qyrym95b0mqmhrrg2ysxjngg";
+  };
+
+  propagatedBuildInputs = with python.pkgs; [
+    blinker
+    bugsnag
+    click
+    dropbox
+    fasteners
+    keyring
+    keyrings-alt
+    pathspec
+    Pyro5
+    requests
+    u-msgpack-python
+    watchdog
+  ] ++ stdenv.lib.optionals stdenv.isLinux [
+    sdnotify
+    systemd
+  ];
+
+  makeWrapperArgs = [
```

Comment:
Are you sure this is necessary? `buildPythonApplication` and `buildPythonPackage` should take care of this.

Reply:
IIRC, they are necessary because it shells out to another process, so PYTHONPATH needs to be set in the wrapper

for the Qt args, that's the current best practice using Qt + python

Reply:
Indeed, as far as I can see https://github.com/SamSchott/maestral/blob/v1.1.0/maestral/daemon.py#L454 and below spawns a subprocess that tries to import maestral.daemon again.